### ISSUE TITLE
Let's address the elephant in the room...

### ISSUE THREAD

**Issue by krahma9990** [efroemling/ballistica#523]
Hello there, Ballistica's community.
I hope everyone is having a wonderful day.

Many people have previously addressed this issue here (At least 3 as far as I know),
The Arabic and Arabic-Like Languages Problem...

Solutions such as switching from image-based to REAL fonts were discussed...
Unfortunately, we saw no progress in that area.

I kindly request that Mr. Eric and the developers available take this matter more seriously.
I know you're all over the place with new bugs popping up, fixes for old ones, and game updates.

**But this... this is a serious flaw.**

The last thing I wanted to talk about with you is chat typing. As we all know, if you want to send something, you have to open the console, type whatever you want, then press Enter, then Escape...

Can't the developers provide something like Counter-Strike Chat system?
You press "Y," type what you want, and then press enter...
Easy Piece Lemon Squeezy

Please let me know what you think, and thank you all.

**Comment by Dliwk** [comment]
Hi, I suppose, that happens on test platforms such as Windows/Linux (with no full unicode support)?

Your chat enhancement sounds cool as additional feature, it shouldn't be something hard to do if someone wants to (for those who want: ask Eric or, well, me). The only obstacle I see is that people won't be able to use 'y' as normal input like grab or punch, but we also can update keyboard config window and make things like y, Esc, F5, F7, etc. also customizable.

**Comment by krahma9990** [comment]
> Hi, I suppose, that happens on test platforms such as Windows/Linux (with no full unicode support)?

Yes, that's why a lot of people (including Mr. Eric) proposed that a font based text is better that texture font test



> Your chat enhancement sounds cool as additional feature

Thx :))



> The only obstacle I see is that people won't be able to use 'y' as normal input like grab or punch, but we also can update keyboard config window and make things like y, Esc, F5, F7, etc. also customizable.

Yes, That's not an issue I just wanted to give an example... cuz when we press F5 The big tab show up and it takes like 40% of the screen...



Thanks for replying @Dliwk,
This game is so underrated hope one day it gets the fame it deserves.

**Comment by MatiasMGz** [comment]
Any update?

**Comment by AnewAlgerianboy** [comment]
> Any update?

None...

**Comment by MatiasMGz** [comment]
A sad situation... Even after 2 years and a half, this is still the elephant in the room.

**Comment by AnewAlgerianboy** [comment]
> A sad situation... Even after 2 years and a half, this is still the elephant in the room.

From my pov I see that this is killing a big element in the game, chatting with people is just an important aspect... 
And as you said after 2 years and a half,  still the request is still going unnoticed, while looking into smaller bugs and updating some things that most of the players will most likely unnoticed...

**Comment by MatiasMGz** [comment]
Yeah, maybe its on the low priority list for now. Just sad for people who speak Arabic-like languages.

**Comment by Loup-Garou911XD** [comment]
It is planned for 2.0 release
